### DISCUSSION TITLE
Community QNA

### DISCUSSION THREAD

**Discussion by yashsehgal** [DesignSystemsOSS/eccentrictouch#191] [Ideas]
This is where you can ask all your queries.

**Comment by Nandani-Paliwal** [comment]
can we add gradient in this project ....if yes then Iwould like to raise issue

**Reply by yashsehgal** [reply]
Hey, 👋🏽 Could you please elaborate on this idea? After the discussion we can surely raise an issue regarding this.

**Things to be discussed**
- How it will be like?
- How it will be implemented?
- And some additional ideas, as well

**Comment by Nandani-Paliwal** [comment]
I will be adding the (different types i.,e  radial,linear ,
diagonal)gradient feature in CSS and to look it more interesting we can
make a DS logo (means the colors will be of DS logo) or can also give it a
look of color wheel(like color picker ).

**Reply by WebShivam** [reply]
> 
> I will be adding the (different types i.,e radial,linear , diagonal)gradient feature in CSS and to look it more interesting we can make a DS logo (means the colors will be of DS logo) or can also give it a look of color wheel(like cold picker ).
> […](#)
> On Wed, Jun 9, 2021, 10:15 AM Yash Sehgal ***@***.***> wrote: Hey, 👋🏽 Could you please elaborate on this idea? After the discussion we can surely raise an issue regarding this. *Things to be discussed* - How it will be like? - How it will be implemented? - And some additional ideas, as well — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#191 (reply in thread)](https://github.com/DesignSystemsOSS/eccentrictouch/discussions/191#discussioncomment-843330)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AUATHGRMAY3EJW22MQOCXHTTR3WVPANCNFSM45J6AW6Q> .

Which part of the project you're thinking to apply this

**Reply by yashsehgal** [reply]
> I will be adding the (different types i.,e radial,linear , diagonal)gradient feature in CSS and to look it more interesting we can make a DS logo (means the colors will be of DS logo) or can also give it a look of color wheel(like color picker ).
> […](#)
> On Wed, Jun 9, 2021, 10:15 AM Yash Sehgal ***@***.***> wrote: Hey, 👋🏽 Could you please elaborate on this idea? After the discussion we can surely raise an issue regarding this. *Things to be discussed* - How it will be like? - How it will be implemented? - And some additional ideas, as well — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#191 (reply in thread)](https://github.com/DesignSystemsOSS/eccentrictouch/discussions/191#discussioncomment-843330)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AUATHGRMAY3EJW22MQOCXHTTR3WVPANCNFSM45J6AW6Q> .

We can surely work on this @Nicks-tech 👍🏽 
We can create a set of class-based properties (utility classes). And according to the theme, we can design these gradients.

Also, @Nicks-tech, do you have access to the Figma file where we have these components, most probably, we will be working there only, regarding this feature. ⭐

**Reply by Nandani-Paliwal** [reply]
yeah..I have acess to it but I don't know how to use figma.....Will  try to learn it and if there is some resource to learn it then plz share...I have created my account there.
   Also ,May I raise isuue now? @yashsehgal

**Reply by Nandani-Paliwal** [reply]
Srry @WebShivam but didn't get ur point...could you plzz elaborate.

**Reply by yashsehgal** [reply]
> yeah..I have acess to it but I don't know how to use figma.....Will try to learn it and if there is some resource to learn it then plz share...I have created my account there.
> Also ,May I raise isuue now? @yashsehgal

Also, as we are having a personal google meet tomorrow, we can master all these ideas/concepts there as well. 

Will see you there :D

**Reply by Nandani-Paliwal** [reply]
ok....will see you then

**Reply by WebShivam** [reply]
> yeah..I have acess to it but I don't know how to use figma.....Will try to learn it and if there is some resource to learn it then plz share...I have created my account there.
> Also ,May I raise isuue now? @yashsehgal

These two resources can be superhelpful for getting started with Figma  @Nicks-tech
Do Check it out if possible, 
https://www.youtube.com/watch?v=Gu1so3pz4bA
https://www.youtube.com/watch?v=4W4LvJnNegA

**Comment by Nandani-Paliwal** [comment]
Thank you😊
